azure-pipelines/createBuildDirectories.sh

```diff
@@ -39,35 +42,12 @@ dotnetProjectArray=(
 framework7="/bin/Debug/net7.0/"
 
 requiredLocDirectories=(
-    "cs"
-    "cs-CZ"
```

Comment:
Why are we removing others?

Reply:
Those do not have the casing problem. Only the 3 langs have it pr-BR, zh-hans, zh-hant